pkg/tsdb/loki/streaming.go

```diff
@@ -13,8 +13,6 @@ import (
 	"github.com/gorilla/websocket"
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
-
-	"github.com/grafana/grafana/pkg/services/featuremgmt"
 )
 
 func (s *Service) SubscribeStream(_ context.Context, req *backend.SubscribeStreamRequest) (*backend.SubscribeStreamResponse, error) {
```

Comment:
Is this *ONLY* called when lokiLive feature toggle is enabled?  If so this is fine, but I thought we were trying to do both?

Reply:
hi @ryantxu ... i'll try to summarize the situation here to be sure there are no misunderstandings:
(note: this functionality has nothing to do with the "tail" functionality in explore-mode)

1. the `lokiLive` feature flag adds the `mode=streaming` option into the loki query editor. when that mode is chosen, the request comes here, to `loki/streaming.go`
2. in `loki/streaming.go` there are two ways to communicate with Loki: the "stable"  `/loki/api/v1/tail` endpoint, and the experimental `/loki/api/v2alpha/tail` endpoint. the system chooses between them based on the `lokiDataframeApi` feature-flag.

i talked with @slim-bean about how to move forward with these experimental feature-flags, and got the info that the combination of `lokiLive=true & lokiDataframeApi=false` combination does not make sense (meaning, we stream, but go to the old endpoint). based on this info, i removed the feature-flag `lokiDataframeApi`, and everywhere i assumed it is set to `true`. this is the PR we are looking at.

is this acceptable?

(in general, we can always add code back, i'm just trying to move somehow forward with these experimental feature-flags. if you have a different idea, i'm fine with discussing it 👍 )